samples/apps/autogen-assistant/README.md

```diff
@@ -0,0 +1,87 @@
+# AutoGen Research Assistant
+
+![ARA](./docs/ara_stockprices.png)
+
+AutoGen Research Assistant is an Autogen-powered AI research assistant that can converse with you to help you conduct research, write and execute code, run saved skills, learn new skills by demonstration, and adapt in response to your interactions.
+
+Project Structure:
+
+- _autogenra/_ code for the backend classes and web api (FastAPI)
+- _frontend/_ code for the webui, built with Gatsby and Tailwind
+
+## Getting Started
+
+Install requirements:
+
+```bash
+pip install -e .
+```
+
+AutoGen requires access to an LLM. Please see the [AutoGen docs](https://microsoft.github.io/autogen/docs/FAQ#set-your-api-endpoints) on how to configure access to your LLM. We recommend setting up your `OPENAI_API_KEY` environment variable first and `OAI_CONFIG_LIST` environment variable to contain a list of OpenAI model configurations.
+
+```bash
+export OPENAI_API_KEY=<your_api_key>
+```
+
+### Build and Run Frontend UI
+
+Note that your Gatsby requires node > 14.15.0 . You may need to [upgrade your node](https://stackoverflow.com/questions/10075990/upgrading-node-js-to-latest-version) version as needed.
+
+```bash
+npm install --global yarn
+cd frontend
+yarn install
+yarn build
+```
+
+The command above will build the frontend ui and copy the build artifacts to the `autogenra` web ui folder. Note that you may have to run `npm install  --force --legacy-peer-deps` to force resolve some peer dependencies.
+
+Run the web ui:
+
+```bash
+autogenra ui --port 8081
```

Comment:
Everything went fine until this line: "bash: autogenra: command not found". Am I missing something?

Reply:
That response suggests your pip install did not work correctly.
Can you confirm the output of 

```python 
pip install -e . 
``` 

The line above should install the sample app package as python module available on your path.

Reply:
Thank you for your help. After `pip install -e .` it worked.